index.js

```diff
@@ -28,11 +33,37 @@ module.exports = function (str, cols) {
 		}
 
 		if (++visible >= cols - 2 && lastSpace > 0) {
-			ret = spliceString(ret, lastSpace, 1, '\n');
+			pre = spliceString(pre, lastSpace, 1, '\n');
 			lastSpace = 0;
 			visible = 0;
 		}
 	}
 
+	visible = 0;
+	for (var j = 0; j < pre.length; j++) {
+		var y = pre[j];
+		ret += y;
+		if (y === '\u001b') {
```

Comment:
@Qix- I'm not familiar with \u009b and didn't notice it elsewhere. What does it represent?